File: Suma de potencias de x/suma_potencias_x_v1.c
/* Creado por Andrés Melgar
 * para iterando++ http://iterando.online/
 * entrada: 
 */
#include <stdio.h>
#include <math.h>

int main() {
    double x;
    int n;
    printf("Ingrese el valor de x(|x|<1): ");
    scanf("%lf", &x);
    printf("Ingrese la cantidad de términos: ");
    scanf("%d", &n);

    if (fabs(x) >= 1)
        printf("Valor de x incorrecto\n");
    else {
        int i = 0;
        double suma = 0;
        while (i < n) {
            suma += pow(x, i);
            i++;
        }
        printf("El valor de 1/(1-x) es: %lf\n", suma);
    }
    return 0;
}

File: Suma de potencias de x/suma_potencias_x_v1.py
#Creado por Andrés Melgar
#para iterando++ http://iterando.online/
#entrada: http://iterando.online/ejercicios-resueltos/suma-de-potencias-de-x-en-python/
x=float(input("Ingrese el valor de x(|x|<1): "))
n=int(input("Ingrese la cantidad de términos: "))

if abs(x) >= 1:
    print("Valor de x incorrecto")
else:
    i=0
    suma=0
    while i<n:
        suma += x**i
        i+=1
    print("El valor de 1/(1-x) es: {:.3f}".format(suma))
